Week02/Assignment/jtripathy_01.py

```diff
@@ -0,0 +1,99 @@
+
+# coding: utf-8
+
+# In[35]:
+
+
+import numpy as np
+import matplotlib  
+import matplotlib.pyplot as plt 
+from IPython.display import Image 
+from numpy import trapz
+from scipy.integrate import simps
+import scipy, pylab
+
+
+x = np.array([ 1.,   1.5,  2.,   2.5,  3.,   3.5,  4.,   4.5,  5.,   5.5,  6.,   6.5,  7.,   7.5, 8.,   8.5,  9.,   9.5, 10. ])
+y = np.array([3.43, 4.94, 6.45, 9.22, 6.32, 6.11, 4.63, 8.95, 7.8, 8.35, 11.45, 14.71, 11.97, 12.46, 17.42, 17.0, 15.45, 19.15, 20.86])
+# y = y.astype('float64')
+plt.subplot(2,1,1)
+plt.scatter(x,y)
+fit_1 = np.polyfit(x,y,1)
+fit_lin = np.poly1d(fit_1) 
```

Comment:
The assignment requests that you use the `curve fit` function to fit the cubic. It shouldn't be too difficult, so could you go back and change this to use that function?